README.md

```diff
@@ -1034,6 +1036,23 @@ Example configuration of `no_log` parameter
     no_log: 'true'
 ```
 
+#### Auto upgrade
+With this parameter you can influence the behaviour during an operator upgrade.  
+If set to true the operator will upgrade the specific instance directly.  
+When the value is set to false and we have a running deployment the operator will not update the AWX instance.  
+This can be usefull when you have multiple AWX instances which you want to upgrade step by step instead all at once.  
+
+
+| Name         | Description                        | Default |
+| -------------| ---------------------------------- | ------- |
+| auto_upgrade | Automatic upgrade of AWX instances | 'true'  |
+
+Example configuration of `auto_upgrade` parameter
+
+```yaml
+  spec:
+    auto_upgrade: 'true'
```

Comment:
Is this intentionally represented as a string here?

Reply:
Nope. Change now.